aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java

```diff
@@ -502,15 +501,7 @@ public void testMultipleOfflineMutations() {
             assertNotNull(postId);
 
             int numberOfLatches = 3;
-            new Thread(() -> {
-                // TODO: What on earth is this business?
-                for (int i = 0; i < numberOfLatches; i++) {
-                    Wifi.turnOff();
-                    Sleep.seconds(3);
-                    Wifi.turnOn();
-                    Sleep.seconds(7);
-                }
-            }).start();
+            Wifi.turnOff();
```

Comment:
The JavaDoc above the test says that WiFi is supposed to be off. So? _Just Do It™._

Reply:
I guess they must somehow start the emulator without cell service and toggle online state by manipulating the Wifi.

Reply:
I'm not sure about this either! I don't see any reason why the emulator wouldn't have cellular data. Not sure what the design was, here.